config/config.go

```diff
@@ -32,6 +32,7 @@ type AppConfig struct {
 	AppURL            string               `yaml:"app_url"`
 	Email             EmailConfig          `yaml:"email"`
 	DataPolicyURL     string               `yaml:"data_policy_url"`
+	MaxTeamMembers    string               `yaml:"MaxTeamMembers"`
```

Comment:
I'd say a better name for the field would be `SoftMaxTeamMembers` since it is the soft limit. `MaxTeamMembers` implies a hard limit.

Also could you please use snake_case for yaml names like the other AppConfig fields? (soft_max_team_members)